scripts/managedb/csubs_control/csub_dups_checker.py

```diff
@@ -100,12 +100,22 @@ def log_status_restart():
     n += 1
     csub_id = str(doc['_id'])
 
-    # The following fields are removed from signature calculation: _id, lastNotification, count and expiration
-    # (not sure about throttling... as I have the doubt, I'm leaving it byt the moment)
+    # The following fields are removed from signature calculation: 
+    #
+    # - _id, 
+    # - lastNotification
+    # - count 
+    # - status
+    # - expiration
+    # - description
+    # 
+    # (Not sure about throttling... as I have the doubt, I'm leaving it by the moment)
     doc.pop('_id', None)
     doc.pop('lastNotification', None)
     doc.pop('count', None)
+    doc.pop('status', None)
     doc.pop('expiration', None)
+    doc.pop('description', None)
```

Comment:
I would use a try-except in all pop because if same field does not exist an error is returned,ex:
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
KeyError: 'description'
```